sale_commission/security/ir.model.access.csv

```diff
@@ -14,3 +14,4 @@ access_sale_commission_settlement_line_manager,access_sale_commission_settlement
 access_sale_commission_settlement_line_salesman,access_sale_commission_settlement_line_salesman,model_sale_commission_settlement_line,sales_team.group_sale_salesman,1,0,0,0
 access_sale_commission_settlement_line_user,access_sale_commission_settlement_line_user,model_sale_commission_settlement_line,base.group_user,1,0,0,0
 access_sale_commission_analysis_report,access_sale_commission_analysis_report,model_sale_commission_analysis_report,base.group_user,1,0,0,0
+access_sale_order_commission_analysis_report,access_sale_order_commission_analysis_report,model_sale_order_commission_analysis_report,base.group_user,1,0,0,0
```

Comment:
Shouldn't be sale user?

Reply:
Modified to `sales_team.group_sale_salesman`